composer.json

```diff
@@ -10,19 +10,19 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": ">=7.0",
-        "symfony/yaml": "^3.0|^4.0|^5.0",
-        "symfony/console": "^3.0|^4.0|^5.0",
-        "symfony/finder": "^3.0|^4.0|^5.0",
-        "symfony/process": "^3.0|^4.0|^5.0",
-        "nikic/php-parser": "~3.0",
-        "gitonomy/gitlib": "0.1.*@dev",
+        "php": ">=7.2",
+        "ext-json": "*",
+        "symfony/yaml": "^5.0",
```

Comment:
These symfony's should be changed to `^5.0 || ^6.0"`

Might as well also do `"composer/composer": "^1.5 || ^2.0"` and `"phpunit/phpunit": "^8.0 || ^9.0"`

Reply:
Done :)